contracts/UpkeepRegistrationRequests.sol

```diff
@@ -95,7 +103,8 @@ contract UpkeepRegistrationRequests is Owned {
       external
       onlyLINK()
     {
-        bytes32 hash = keccak256(msg.data);
+        require(adminAddress != address(0), "invalid admin address");
+        bytes32 hash = keccak256(abi.encode(upkeepContract, gasLimit, adminAddress, checkData));
 
         emit RegistrationRequested(
```

Comment:
It would be nice to move this event to the end, so we could get a summary of how much funding has been added total instead of just in this request, but I can see that being a pretty niche use case. Not necessary, but maybe nice to have.

Reply:
@se3000 Doing this might be a little clunky because of the auto approve feature. With that enabled, requests aren't saved and don't accumulate a balance. What might be more useful here is a getter, something like `getPendingRequest(hash)` that returns the balance & admin of that request.